backend/src/api/api_v1/endpoints/admin.py

```diff
@@ -0,0 +1,72 @@
+from typing import Any
+
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.ext.asyncio import AsyncSession
+from src import crud
+from src.api import deps
+from src.core.config import settings
+from src.db.db import get_session
+from src.models.user import User, UserCreate, UserResponse, UserUpdate
+from src.utils import send_new_account_email
+
+router = APIRouter()
+
+@router.delete("/{user_id}", response_model=UserResponse)
+async def remove_user(
+    user_id: int,
+    db: AsyncSession = Depends(get_session),
+    current_user: User = Depends(deps.get_current_active_admin),
+) -> Any:
+    """
+    Remove an existing user. Must be logged in first. Must be an admin.
+    """
+    user = await crud.user.get(db, id=user_id)
+
+    if user is None:
+        raise HTTPException(status_code=404, detail="That user does not exist.")
+
+    if user.user_id != current_user.id:
```

Comment:
remove this case, admins should be able to delete any user as long as that user is not an admin